tests/tasks/create_test_interfaces_with_dhcp.yml

```diff
@@ -43,8 +43,15 @@
       nmcli d set testbr managed false
     fi
     ip link set testbr up
-    ip addr add 192.0.2.1/24 dev testbr
-    ip -6 addr add 2001:DB8::1/32 dev testbr
+    timer=0
+    while ! ip a s testbr | grep -q 'inet [1-9]'
```

Comment:
IMHO it would be better not to use abbreviations for the ip command.

Reply:
I changed into `while ! ip addr show testbr | grep -q 'inet [1-9]'` to be consistent.